addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -3883,6 +3883,18 @@ options.registry.layout_column.include({
     _computeVisibility() {
         return !this.$target[0].closest('[data-snippet="s_images_wall"]');
     },
+    /**
+     * @override
+     */
+    _getTopOffset() {
+        let offset = this._super(...arguments);
+        // Get the header height if it is not a sidebar.
+        const headerEl = this.$target[0].ownerDocument.body.querySelector("header#top");
+        if (!headerEl.classList.contains("o_header_sidebar")) {
+            offset += headerEl.offsetHeight;
+        }
+        return offset;
```

Comment:
Great that you thought of the nav bar :+1: But it's possible that the header has an offsetHeight at the same time as being invisible no ? It should not be considered in this case right ?

Reply:
Indeed, I will consider the header bottom position instead of the height then, it should cover all the cases :slightly_smiling_face: